<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicExample</class>
 <widget class="QWidget" name="MusicExample">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButtonLoadMusic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Music</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSaveMusic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save to XML</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_play">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_stop">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>81</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_volume">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>20</width>
     <height>71</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_drum">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="maxVisibleItems">
    <number>6</number>
   </property>
   <item>
    <property name="text">
     <string>No Rhythm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 5</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_chineseBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="maxVisibleItems">
    <number>6</number>
   </property>
   <item>
    <property name="text">
     <string>No Rhythm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 5</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_triangle">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="maxVisibleItems">
    <number>6</number>
   </property>
   <item>
    <property name="text">
     <string>No Rhythm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 5</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_congas">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="maxVisibleItems">
    <number>6</number>
   </property>
   <item>
    <property name="text">
     <string>No Rhythm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 5</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_Tamborine">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="maxVisibleItems">
    <number>6</number>
   </property>
   <item>
    <property name="text">
     <string>No Rhythm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 5</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_beatBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="maxVisibleItems">
    <number>6</number>
   </property>
   <item>
    <property name="text">
     <string>No Rhythm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rhythm 5</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Drum</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Chinese box</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Triangle</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Congas</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tamborine</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Beat box</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_instrument">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Piano</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Flute</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Violin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Xylophone</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Trumpet</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButtonSaveMusicWav">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save to Wav</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSaveMusicMp3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save to Mp3</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
